django/db/backends/base/schema.py

```diff
@@ -49,12 +49,16 @@ def _related_non_m2m_objects(old_field, new_field):
     related_fields = zip(
         (
             obj
-            for obj in _all_related_fields(old_field.model)
+            for obj in sorted(
+                _all_related_fields(old_field.model), key=lambda f: f.name
```

Comment:
`_all_related_fields` is used only in `_related_non_m2m_objects` so I would move `sorted` there:
```python
def _all_related_fields(model):
    return sorted(model._meta._get_fields(
        forward=False,
        reverse=True,
        include_hidden=True,
        include_parents=False,
))
```
We also should add a comment why `sorted()` is called to avoid removing it in the future, maybe:
```
# Related fields must be returned in deterministic order.
```